src/progress_bar_exercise/ProgressBarExercise.js

```diff
@@ -17,6 +20,47 @@ export default ProgressBarExercise;
 
 // ----------------------------------------------------------------------------------
 
+const strings = {
+  LOADING: "LOADING",
+  START_REQUEST: "START REQUEST",
+  FINISH_REQUEST: "FINISH REQUEST",
+};
+
+const divContainerStyle = {
+  display: "flex",
+  justifyContent: "center",
+  gap: "20px",
+  padding: "20px",
+  margin: "20px",
+};
+
 const Solution = () => {
-  return <div>Add solution here</div>;
+  const { isLoading, isFinishing, startRequest, finishRequest } =
+    useSimulateRequest();
+  const [showProgressBarV2, setShowProgressBarV2] = useState(false);
+
+  return (
+    <div>
+      <ProgressBar
+        isLoading={isLoading}
+        isFinishing={isFinishing}
+        breakpoints={[20, 40, 60, 90]}
```

Comment:
Currently breakpoints are very manual and may be prone to developer error. I would like to add more safety here for handling bad inputs (not numbers, numbers out of range, numbers out of order, etc)